app/src/org/commcare/android/javarosa/IntentCallout.java

```diff
@@ -319,6 +312,7 @@ private void storePointerToFileResponse(TreeReference ref, String responseValue,
         } catch (IOException ioe) {
             ioe.printStackTrace();
             Log.e(TAG,"CommCare failed to copy a binary input from an intent callout from: " + responseValue);
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dimagi/commcare-android/blob/998b1c581d99eb009f0de7fd37ea42e538db17c5/app/src/org/commcare/android/javarosa/IntentCallout.java#L314"></span>Using 'Log' instead of 'Timber'</td>
    </tr>
  </tbody>
</table>